packages/create-nuxt3-app/src/actions.ts

```diff
@@ -0,0 +1,28 @@
+import {NodePlopAPI} from "node-plop"
```

Comment:
can you please setup & congigure & run eslint with this package, so that all files are correclty formatted?